java/runtime/src/main/java/org/ray/runtime/object/RayObjectSerializer.java

```diff
@@ -0,0 +1,39 @@
+package org.ray.runtime.object;
+
+import org.nustaq.serialization.FSTBasicObjectSerializer;
+import org.nustaq.serialization.FSTClazzInfo;
+import org.nustaq.serialization.FSTObjectInput;
+import org.nustaq.serialization.FSTObjectOutput;
+import org.ray.api.id.ObjectId;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class RayObjectSerializer extends FSTBasicObjectSerializer {
```

Comment:
I don't think this serializer is necessary. I will create another PR to remove `NativeRayActorSerializer` too.

Reply:
It turns out that `NativeRayActorSerializer` is needed for cross-language actor handle serialization. But we don't need it for RayObject. So can you try to remove this class and test it?

Reply:
Maybe. I'll try.

Reply:
did this in [6ee1c94](https://github.com/ant-tech-alliance/ray/commit/6ee1c947c8e7b2f958d643f00221cf52d30a7241)